assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -672,7 +681,7 @@ const MapDisplay = ({
     >
       <MapSafeAreaContext.Provider
         value={{
-          paddingTopLeft: [445, 54],
+          paddingTopLeft: [shouldOffset ? 445 : 54, 54],
```

Comment:
**suggestion**: If `MapPage` ever changes the size of it's panels, we'll have to also remember to update these values, in the case that `MapPage` needs to control the safe area, what do you think about making the safe area provided via a prop?

I think it'd also be simpler for testing (not that we've got many tests for this) because instead of asserting it's either 54 or 445, we can assert that the value updates when the prop updates.

Reply:
Are you suggesting that the value of `value` should be passed in as a prop instead of `shouldOffset`? Or that in some way, we could pass the entire provider in?

Reply:
That was what I was suggesting, but your comment made me realize that we could pull `MapSafeAreaContext.Provider` up into `MapPage` containing `MapDisplay` instead of needing to pass args into `MapDisplay` or manage any composition within `MapDisplay`.

Happy to walk through this together in a huddle or provide a code sample if you'd like 👍🏻.

Reply:
Addressed in https://github.com/mbta/skate/pull/2286/commits/dfa22163d6a2ea3dcafd04fc49c92549c5bd977c